appveyor.approval.yml

```diff
@@ -0,0 +1,18 @@
+version: '{build}'
+
+environment:
+    PATH: 'C:\Ruby200\bin;%PATH%'
+
+install:
+    - cmd: bundle install
+
+build_script:
+    - cmd: build.netstd.cmd approve

```

Comment:
Agreed. But if AppVeyor can't send status updates with different contexts, the point is moot...